src/Components/FilterForm/FilterForm.tsx

```diff
@@ -178,13 +187,18 @@ export default function FilterForm() {
           <FinancialPreference text="Oczekiwane wynagrodzenie miesięczne netto (pln)" />
           <InternshipPreference text="Zgodan na odbycie bezpłatnych praktyk/stażu na początek" />
           <Experience text="Ilość miesięcy doświadczenia komercyjnego w programowaniu" />
-          <Button
+          <PreferenceButton
             text="Anuluj"
             color="#0a0a0a"
             className="cancel-btn"
             handleClick={handleCancel}
           />
-          <Button text="Pokaż wyniki" color="#E02735" className="submit-btn" />
+          <Button
+            text="Pokaż wyniki"
+            color="#E02735"
```

Comment:
uuttColor of button is defined in global variable. Consider use this. It makes theme color changeing easier for whole project.